.graphqlconfig

```diff
@@ -1,5 +1,5 @@
 {
-  "schemaPath": "libraries/graphql_schemas/schema.graphql",
+  "schemaPath": "schema.graphql",
```

Comment:
sorry, my bad, `eslintIgnore` not needed, `schema.graphql` was fine :sweat_smile: